### ISSUE TITLE
[EN][Profession][Mining]Pyrite ore is not a rare spawn as it is meant to be

### ISSUE THREAD

**Issue by sunshineinabox** [Atlantiss/BugTracker#3241]
Currently You can see pyrite spawns are very common. Pyrite should act like Titanium and be a rare spawn from Elementium Veins instead of being on its own. Titanium is sometimes spawned when a saronite vein is mined and during respawn the vein has a chance to respawn as titanium. For Pyrite however it seems that some veins are always pyrite. There are also locations where you see Elementium and Pyrite stacked. I think all existing Pyrite veins should be turned into Elementium and make it so that when those are mined there is a "rare" chance that pyrite is spawned in place of it. 

In the Comment section of the wowhead you can see that someone mentions it:

"This is a Khorium ore or Titanium ore for Cataclysm, which is used for highest level item crafting. Can be found in Twilight Highlands and Tol Barad. This is a rare spawn for Elementium ore."

http://www.wowhead.com/item=52183/pyrite-ore#comments

In the Elementium Vein there are also some comments that might help decide on a rate.

http://www.wowhead.com/object=202738/elementium-vein#comments

It seems that for every 10-12 stacks of elementium you can get 2-4 stacks of pyrite. 

There might be more comments with more precise data. I was only able to scan these articles for them.

**Comment by Foreverloved** [comment]
I agree. I've been farming for a while and I might end up with roughly the same amount of pyrite as i do elementium while farming.

**Comment by Thefakemccoy** [comment]
Ive mined one to find the other under it. I guess it no worse than having nodes in mid air.
